plugin/trino-teradata/src/test/java/io/trino/plugin/teradata/integration/clearscape/ClearScapeManager.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.teradata.integration.clearscape;
+
+import io.airlift.log.Logger;
+import io.trino.plugin.teradata.integration.TeradataTestConstants;
+
+import java.net.URISyntaxException;
+import java.util.regex.Pattern;
+
+public class ClearScapeManager
+{
+    private static final Logger log = Logger.get(ClearScapeManager.class);
+    private static final Pattern ALLOWED_URL_PATTERN =
+            Pattern.compile("^(https?://)(www\\.)?api.clearscape.teradata\\.com.*");
+    private Model model;
+
+    private boolean isValidUrl(String url)
+    {
+        return ALLOWED_URL_PATTERN.matcher(url).matches();
+    }
+
+    private TeradataHttpClient getTeradataHttpClient()
+            throws URISyntaxException
+    {
+        String envUrl = TeradataTestConstants.ENV_CLEARSCAPE_URL;
+        if (isValidUrl(envUrl)) {
+            return new TeradataHttpClient(envUrl);
+        }
+        else {
+            throw new URISyntaxException(envUrl, "Provide valid environment URL");
+        }
+    }
+
+    public void init(Model model)
+    {
+        this.model = model;
+    }
```

Comment:
Can we add a constructor with `Model` instead?

Reply:
Yes. Added constructor and removed init